uco-action/action.ttl

```diff
@@ -46,7 +46,7 @@
     rdfs:label "ActionArgument"@en;
     rdfs:subClassOf <http://unifiedcyberontology.org/core#Facet>, [
       rdf:type owl:Restriction ;
-      owl:onProperty action:name ;
+      owl:onProperty action:argumentName ;
```

Comment:
This change would be in error.
'name' is the appropriate property here but there is an issue with the prefix for 'name' and 'value' in the restrictions for ActionArgument. They should each use the core namespace prefix (which should be added to action.ttl) rather than the action namespace prefix.
I have a separate commit addressing this issue.

Reply:
![image](https://user-images.githubusercontent.com/9854293/57554115-665f0c00-7325-11e9-947c-8d2efb9771d4.png)
This change would not be an error, it corrects the error of a inconsistency between a restriction and the class definition. If the intent is to use `name`, then the class definition needs to be changed accordingly.

The class definition for ActionArgument declares the class has 'argumentName' not 'name'.

Reply:
ugh. sorry for the confusion. I was looking at the class definition for ActionArgument which does not declare an 'argumentName'. However, I did not notice that there is an argumentName property defined with ActionArgument as its domain. So, you are correct, from Protege's perspective it is part of the class definition. These are still not the way those properties were supposed to be defined. Not sure how there ended up with local redefinitions of name (argumentName) in this case and value properties in action.ttl when they should just use the already defined name and value in core. Once again, I apologize for the confusion and asserting that this proposed change was in error.

Reply:
No problem. There are domain and range assertions in the current UCO ontolgy that will cause problems such as this. Ferreting them out and correcting them is on my "to do list". However, I'm pushing us to get the obvious errors corrected first. This was one of them. Correcting this one to reflect the desired core:name is not a problem now that I know the intent. But... that means that the core:name predicate should not have a domain set. Otherwise, an inference engine will infer ANY object with a core:name set will be of the set domain types.

Reply:
@sbarnum I will have to verbally go over this with you. There is a DataProperty observable:name, and an ObjectProperty core:name.